pkg/plugins/autodiscovery/githubaction/main.go

```diff
@@ -163,6 +167,10 @@ func New(spec interface{}, rootDir, scmID string) (GitHubAction, error) {
 		newFilter.Kind = defaultVersionFilterKind
 		newFilter.Pattern = defaultVersionFilterPattern
 	}
+	digest := false
```

Comment:
```suggestion
	digest := true
```

What would you think to enable digest by default? 
With the way comments are handled and It think it's the best of the two worlds

Reply:
Keeping the none digest behavior by default for now